docs/email/patterns.html

```diff
@@ -0,0 +1,71 @@
+            <div class="grid--cell mr8 fc-yellow-600">
+                <svg aria-hidden="true" class="svg-icon iconAlert" width="18" height="18" viewBox="0 0 18 18"><path d="M7.95 2.71c.58-.94 1.52-.94 2.1 0l7.69 12.58c.58.94.15 1.71-.96 1.71H1.22C.1 17-.32 16.23.26 15.29L7.96 2.7zM8 6v5h2V6H8zm0 7v2h2v-2H8z"></path></svg>
+            </div>
+            <div class="grid--cell" aria-label="notice-message">
+                <p class="m0">
+                    An email should only contain content from <strong>one Team</strong>. A Team's content should never appear mixed alongside with content from another Team or public content in a single email.
+                </p>
+            </div>
+        </div>
+    </aside>
+    <div class="stacks-preview">
+{% highlight html linenos %}
+<table border="0" cellpadding="0" cellspacing="0" role="presentation" align="left">
+    <tr>
+        <td style="font-family: arial, sans-serif; font-size: 15px; line-height: 15px; color: #ffffff; background-color: #EB5757; font-weight: bold; text-align: center; padding: 4px 6px; border-radius: 3px; vertical-align: middle;">
+            T
+        </td>
+        <!-- Optional table cell for Team name : BEGIN -->
+        <td style="padding-left: 5px; font-family: arial, sans-serif; font-size: 15px; line-height: 15px; color: #EB5757; font-weight: bold; text-align: left; vertical-align: middle;">
+            Team Name
+        </td>
+        <!-- Optional table cell for Team name : END -->
+    </tr>
+</table>
+{% endhighlight %}
+        <div class="stacks-preview--example" style="font-family: arial, sans-serif; font-size: 15px; line-height: 19px; color: #54595F; text-align: left;">
+            <table border="0" cellpadding="0" cellspacing="0" role="presentation">
+                <tr>
+                    <td style="font-family: arial, sans-serif; font-size: 15px; line-height: 15px; color: #ffffff; background-color: #EB5757; font-weight: bold; text-align: center; padding: 4px 6px; border-radius: 3px; vertical-align: middle;">
+                        T
+                    </td>
+                    <td style="padding-left: 5px; font-family: arial, sans-serif; font-size: 15px; line-height: 15px; color: #EB5757; font-weight: bold; text-align: left; vertical-align: middle;">
+                        Team Name
+                    </td>
+                </tr>
+            </table>
+            <br>
+            <table border="0" cellpadding="0" cellspacing="0" role="presentation">
+                <tr>
+                    <td style="font-family: arial, sans-serif; font-size: 15px; line-height: 15px; color: #ffffff; background-color: #EB5757; font-weight: bold; text-align: center; padding: 4px 6px; border-radius: 3px; vertical-align: middle;">
+                        T
+                    </td>
+                </tr>
+            </table>
+        </div>
+        <aside class="s-notice s-notice__info s-notice__outlined mt24" role="status" aria-hidden="false">
```

Comment:
This aside should sit outside of `div.stacks-preview--example`. It currently appears to be apart of it:

![image](https://user-images.githubusercontent.com/1544281/43609349-bebdee48-9671-11e8-8409-2b3714525b24.png)